include/si.h

```diff
@@ -16,7 +16,7 @@ typedef s32 ScriptID;
 #define SI_SAVE_VAR(v) (v - 170000000)
 #define SI_ARRAY(v) (v - 190000000)
 #define SI_ARRAY_FLAG(v) (v - 210000000)
-#define SI_FIXED(v) ((s32)(v * 1024.0f) - 230000000) // See float_to_fixed_var
+#define SI_FIXED(v) ((v * 1024.0f) + -230000000) // See float_to_fixed_var
```

Comment:
wat

Reply:
Changing to this allows me to call SI_FIXED and match a function, but I can just leave this alone and not use it in that function if you prefer.

Reply:
No, it's fine, I was just wondering why

Reply:
gorcha